apis/python/src/tiledbsoma/_sparse_nd_array.py

```diff
@@ -105,16 +105,16 @@ def read(
         partitions: Optional[options.ReadPartitions] = None,
         platform_config: Optional[PlatformConfig] = None,
     ) -> "SparseNDArrayRead":
-        """Reads a user-defined slice of the SparseNDArray.
+        """Reads a user-defined slice of the :class:`SparseNDArray`.
```

Comment:
given that we’re in the SparseNDArray class itself I’m not sure the cross-reference is that useful here.